pkg/test/testenv.go

```diff
@@ -268,7 +275,7 @@ func prepareCRDs() error {
 		return err
 	}
 
-	cassOperatorCrd := "github.com/k8ssandra/cass-operator/config/crd?ref=" + cassOperatorVersion
+	cassOperatorCrd := "github.com/burmanm/cass-operator/config/crd?ref=" + cassOperatorVersion
```

Comment:
We need to remove this later, right?

Reply:
I missed that one. I need to update that to point to upstream master.